generators/Common.fs

```diff
@@ -32,86 +28,10 @@ type TestFileFormat =
     | Class
 
 module Logging =
+
+    open Serilog
```

Comment:
Any reason for this `open` being inside the `module` while most other declarations are outside?

Reply:
You are correct that they are currently mixed. I've looked at some code, and I think the guideline is to make the open statements as local as possible: https://github.com/Microsoft/visualfsharp/blob/master/src/fsharp/FSharp.Core/array.fs#L1274

Shall I make the change?

Reply:
Ok let's standardize on that then. Not urgent to make the change but if you'd like to I'd say let's go with another PR.